axlearn/common/env_test.py

```diff
@@ -0,0 +1,17 @@
+"""Tests for AXLearn environment."""
+# pylint: disable=no-self-use,redundant-keyword-arg,too-many-function-args
```

Comment:
OOI where did `redundant-keyword-arg,too-many-function-args` come from?

Reply:
Removed, thanks! They were from another test that I copied over as a template.